grunt_tasks/plugin.js

```diff
@@ -369,16 +370,16 @@ module.exports = function (grunt) {
                                 [`girder_plugins/${plugin}/node`]: pluginNodeDir,
                                 [`girder_plugins/${plugin}`]: webClient
```

Comment:
With this change, it would seem that the current aliasing of the node_modules for plugins here may no longer be necessary, since any plugins the plugin depends on should have been added to the search path. Could these aliases be removed?